api/volume.go

```diff
@@ -2,6 +2,7 @@ package api
 
 import (
 	"fmt"
+	"github.com/rancher/lasso/pkg/log"
 	"net/http"
 	"sort"
```

Comment:
nit: reorganize the layout.

Reply:
Reorganize the file and layout to avoid make confused.